src/main/java/com/shopping_app/shoppingApp/domain/Address.java

```diff
@@ -34,4 +38,52 @@ public class Address extends BaseEntity {
     @JsonIgnore
     @JoinColumn(name = "user_id")
     private User user;
+
+    public String getStreet() {
+        return street;
+    }
+
+    public void setStreet(String street) {
+        this.street = street;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public String getState() {
+        return state;
+    }
+
+    public void setState(String state) {
+        this.state = state;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    public String getPincode() {
+        return pincode;
+    }
+
+    public void setPincode(String pincode) {
+        this.pincode = pincode;
```

Comment:
The getter and setter annotation is already there on the class, then why define these methods again?

Reply:
will remove all getter setter methods once i fix the mapstruct issue